### DISCUSSION TITLE
Mozilla / coqui AI ?

### DISCUSSION THREAD

**Discussion by MentalGear** [coqui-ai/TTS#703] [General Q&A]
Since it's never mentioned, but the repo seem to be forked, I'm wondering how https://github.com/mozilla/TTS relates to this repo?

**Comment by erogol** [comment]
CoquiTTS is a hard fork with continuing development. I've created both of the libraries and now moved here. It is also mentioned in the TTS discourse channel and all the social platforms I have access to when it was forked. 

Hope this helps.

**Comment by MentalGear** [comment]
Thanks maybe add this as a section to nthe raleadme?

**Reply by JacobFV** [reply]
I agree. Making things more transparent in the readme would help developers know where to go when they face issues that are already addressed in the Mozillar repository.

**Reply by thorstenMueller** [reply]
I'm not sure if @erogol still has permission to update README on Mozilla TTS repo. So bringing that info there might be a problem.

**Reply by JacobFV** [reply]
No I mean, coqui AI should update their readme to point back to the ancestor repo (Mozilla)
